### ISSUE TITLE
Using 5 handed tables, starting with 6 and quitting 2 locked up one table

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#1421]
Fix the bug that caused one table to pause while the other played on.

I altered `wsop_2024_98_closer_nlhe` to use 5 handed tables, then started a tournament with 6 players, then quit two on one table. I was thinking it would split to two and two, but I didn't think through the fact that four is less than 5 and that it would then combine to a single table. However, it didn't combine to a single table. It did balance to two and two, then it had one table pause for combining, but the other table ran indefinitely.

This is an artificial condition but it still shouldn't happen. So, once I get #1415 to a stopping point, I should work on this one.

Oy!

**Comment by ctm** [comment]
This also happened using 3 handed tables, starting with 6 and quitting 2. I'll try starting with 5 and quitting 1.

**Comment by ctm** [comment]
Starting with 5 and quitting _2_ reproduced. I'll add instrumentation and see if I can start with 4 and quit 1. FWIW, the hung table does show the

> Pausing to combine to final table...

message (which comes from `PausedForRedraw`).

**Comment by ctm** [comment]
This looks like a problem I introduced when I added a merge delay. It's supposed to detect if one of the tables only has a single player, but it doesn't check both tables and only checks the table with two players. This hasn't come up in a real game, but I'm glad I found it.

I haven't fixed the problem yet. There may be additional problems when getting down to a single player due to quitting, but I need to fix `tables_needed_with_potential_merge_delay`, before I get to them.
